analyzer/src/funTest/assets/projects/synthetic/spdx/project/project.spdx.yml

```diff
@@ -11,6 +11,12 @@ dataLicense: "CC0-1.0"
 documentNamespace: "http://spdx.org/spdxdocs/spdx-document-xyz"
 documentDescribes:
 - "SPDXRef-Package-xyz"
+externalDocumentRefs:
+- externalDocumentId: "DocumentRef-curl-7.70.0"
```

Comment:
I added the version on purpose. In a real life file it would be possible to refer to several versions of the same dependency and this file could be used as a reference file to create ones own SPDX file. I thought it better to use an ID that makes it easy to identify the actual version, also in later use in the relationships.

Reply:
I could add "Package" to the ID, but I checked the SPDX specifications [1]  to see how they did it, and thought we could adhere to their example in our example. They don't specifically add a "what kind of document this refers to" to the identifier. However, this would make it more clear. I am unsure. What do you think? @sschuberth 

[1] https://spdx.github.io/spdx-spec/2-document-creation-information/#26-external-document-references

Reply:
Right, this would just be our personal convention, similar to like we use "SPDXRef-Package-xyz" although we could also use just "SPDXRef-xyz". Personally, I have a *slight* preference to adding "-Package", but I'd also appreciate @tsteenbe's feedback here.